tests/generators/bls/main.py

```diff
@@ -29,6 +30,13 @@ def int_to_hex(n: int, byte_length: int = None) -> str:
     return encode_hex(byte_value)
 
 
+def int_to_bytes(n: int, byte_length: int = None) -> bytes:
+    byte_value = int_to_big_endian(n)
+    if byte_length:
+        byte_value = byte_value.rjust(byte_length, b'\x00')
```

Comment:
I'm concerned about the new null-terminated strings.
I think a comment should be added that it creates null-terminated string
only for the BLS spec (https://github.com/ethereum/eth2.0-specs/issues/1576)

Reply:
Everything is octet strings, which are _not_ null terminated, including the DST.

The zero bytes here are just padding to length (iiuc they are added to the left in big-endian, so they are prepended, not appended, in any case).